<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoMapperDialogBase</class>
 <widget class="QDialog" name="PhotoMapperDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotoMapper</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>455</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="vertexFileLabel">
        <property name="text">
         <string>Vertex File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vertexPathFileLineEdit">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>*.txt *.csv</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseVertexFileButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="edgeFileLabel">
        <property name="text">
         <string>Edge File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edgePathFileLineEdit">
        <property name="placeholderText">
         <string>*.txt *.csv</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseEdgeFileButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="gridFileLabel">
        <property name="text">
         <string>Grid File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gridPathFileLineEdit">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>*.txt *.csv</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseGridFileButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="button_box">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>PhotoMapperDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>PhotoMapperDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
